BASE/src/Microsoft.ApplicationInsights/Extensibility/Implementation/Authentication/ISupportCredentialEnvelope.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Authentication
```

Comment:
I know it's pedantic, but hear me out:
Authentication is what the server (Breeze) does, to check the token and decide what you're allowed to do.
Authorization is what the client (SDK) does to prove who they are.
So, I think this namespace should be `Authorization`.
It's a common-enough confusion that you'll see inconsistencies in a number of places. In some cases it's ambiguous (is the token you get an "authorization token" or an "authentication token"?), so you see people side-step and call it "auth" or "access".